src/Soil/Soil.jl

```diff
@@ -56,18 +56,20 @@ for interactions between components.
 =#
 
 using ClimaLSM
-using UnPack
 using DocStringExtensions
+using LinearAlgebra
```

Comment:
Why do we need LinearAlgebra in this file?

Reply:
Hm, we need LinearAlgebra in rre.jl (for making TridiagonalW stuff) but rre.jl is included in the Soil.jl module. We have all the "using" "import" etc statements in Soil.jl. perhaps not so clear?